src/test/scala/chiselTests/stage/ChiselStageSpec.scala

```diff
@@ -142,4 +146,54 @@ class ChiselStageSpec extends AnyFlatSpec with Matchers with Utils {
     exactly (1, order) should be (Dependency[chisel3.stage.phases.Elaborate])
   }
 
+  behavior of "ChiselStage$ exception handling"
+
+  it should "truncate a user exception" in {
+    info("A generic ChiselException was thrown, not the underlying java.lang.AssertionError")
+    val exception = intercept[java.lang.AssertionError] {
```

Comment:
If this statement is true, then shouldn't you be intercepting a `ChiselException`?

Reply:
Yep, you're right. This was left over from an early iteration of this PR. Will fix in a fixup commit.

Reply:
Fixed in the rebase.